src/v2/guide/migration.md

```diff
@@ -173,68 +173,68 @@ mounted: function () {
 </div>
 {% endraw %}
 
-### `$index` and `$key` <sup>removed</sup>
+### `$index` y `$key` <sup>removido</sup>
 
-The implicitly assigned `$index` and `$key` variables have been removed in favor of explicitly defining them in `v-for`. This makes the code easier to read for developers less experienced with Vue and also results in much clearer behavior when dealing with nested loops.
+Las variables implícitamente asignadas `$index` y `$key` se han eliminado a favor de definirlas explícitamente en `v-for`. Esto hace que el código sea más fácil de leer para los desarrolladores con menos experiencia en Vue y también da como resultado un comportamiento mucho más claro cuando se trata de bucles anidados.
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of these removed variables. If you miss any, you should also see <strong>console errors</strong> such as: <code>Uncaught ReferenceError: $index is not defined</code></p>
+  <h4>Ruta de actualización</h4>
+  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper"> asistente de migración</a> para encontrar ejemplos de estas variables removidas en su código. Si usted se pierde de alguna, también debería ver los <strong>errores de consola</strong> como: <code>Uncaught ReferenceError: $index is not defined</code></p>
 </div>
 {% endraw %}
 
-### `track-by` <sup>replaced</sup>
+### `track-by` <sup>reemplazado</sup>
 
-`track-by` has been replaced with `key`, which works like any other attribute: without the `v-bind:` or `:` prefix, it is treated as a literal string. In most cases, you'd want to use a dynamic binding which expects a full expression instead of a key. For example, in place of:
+`track-by` ha sido reemplazado por `key`, que funciona como cualquier otro atributo: sin el prefijo `v-bind:` o `:`, se trata como una cadena de texto literal. En la mayoría de los casos, desearía utilizar un enlace dinámico que espera una expresión completa en lugar de una clave. Por ejemplo, en lugar de:
 
-``` html
+```html
 <div v-for="item in items" track-by="id">
 ```
 
-You would now write:
+Usted debería escribir:
 
-``` html
+```html
 <div v-for="item in items" v-bind:key="item.id">
 ```
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of <code>track-by</code>.</p>
+  <h4>Ruta de actualización</h4>
+  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper"> asistente de migración</a> para encontrar ejemplos de <code>track-by</code> en su código.</p>
```

Comment:
I think this will render two spaces, let's remove the one between `Ejecute el` and `asistente de migración`

```suggestion
  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> para encontrar ejemplos de <code>track-by</code> en su código.</p>
```